pkgs/applications/backup/nis-one-click-backup/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+, cmake
+, ninja
+, qt6
+, wrapQtAppsHook
+, extra-cmake-modules
+}:
+
+stdenv.mkDerivation rec {
+  pname = "one-click-backup";
+  version = "1.2.2";
+
+  src = fetchFromGitLab {
+    owner = "dev-nis";
+    repo = "nis-one-click-backup-qt";
+    rev = "9a1ca839ca511d55716ffc8898e5d9ec9449ef18"; #version;
+    hash = "sha256-JbgmfKEpKRj/ph8dkW0IRulmAhk6Dwv/1Z4j3hUgPXk=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    ninja
+    wrapQtAppsHook
+    extra-cmake-modules
+  ];
+
+   buildInputs = [
+    qt6.full
+    qt6.qtbase
+  ];
+
+  meta = with lib; {
+    description = "Simple Program to backup folders to an external location by copying them";
+    homepage = "https://gitlab.com/dev-nis/nis-one-click-backup-qt";
+    changelog = "https://gitlab.com/dev-nis/nis-one-click-backup-qt/-/blob/${src.rev}/CHANGELOG.md";
+    license = licenses.gpl3Only;
+    maintainers = with maintainers; [ ];
```

Comment:
[`meta.maintainers` must be set for new packages.](https://github.com/NixOS/nixpkgs/blob/2565ab7b53ae6966bd08e68203a89645961849b3/pkgs/README.md#meta-attributes)

Please read [`/maintainers/README.md`](https://github.com/NixOS/nixpkgs/blob/2565ab7b53ae6966bd08e68203a89645961849b3/maintainers/README.md), and then add your details to [`/maintainers/maintainers-list.nix`](https://github.com/NixOS/nixpkgs/blob/2565ab7b53ae6966bd08e68203a89645961849b3/maintainers/maintainer-list.nix). Commit that as a separate commit, preceding everything related to this package - look at the git history of the latter for the formatting of the commit title.

Then, adjust this line by adding yourself as the maintainer.

Reply:
This should now be implemented with commit [5f21566](https://github.com/NixOS/nixpkgs/pull/287497/commits/5f2156658dc09abc546d38c65ff73661c6c0c075) .